tests/mgbench/benchmark.py

```diff
@@ -144,28 +101,334 @@ def filter_benchmarks(generators, patterns):
     default="/tmp",
     help="directory path where temporary data should " "be stored",
 )
-parser.add_argument("--no-properties-on-edges", action="store_true", help="disable properties on edges")
+parser.add_argument(
+    "--no-properties-on-edges", action="store_true", help="disable properties on edges"
+)
+
 parser.add_argument("--bolt-port", default=7687, help="memgraph bolt port")
+
+parser.add_argument(
+    "--with-authorization",
+    action="store_true",
+    default=False,
+    help="Run each test with authorization",
+)
+
+parser.add_argument(
+    "--warmup-run",
+    action="store_true",
+    default=False,
+    help="Run warmup before benchmarks",
+)
+
+parser.add_argument(
+    "--mixed-workload",
+    nargs=5,
+    type=int,
+    default=[0, 0, 0, 0, 0],
+    help="Run combination that defines mixed workload;"
+    "Pass the positional arguments as values of what percentage of"
+    "write/read/update/analytical queries you want;"
+    "Running --mixed-workload 1000 20 70 10 0"
```

Comment:
The only change here is that you do not need to specify the query, because it will be automatically executed on every query from group. 
```
./benchmark.py --mixed-workload=[1000, 10, 10, 30, 50, 0] pokec/small/group/**query** 

./benchmark.py --mixed-workload=[1000, 30, 0, 0, 0, 70] pokec/small/group/**query**
```